Streetcode/Streetcode.XUnitTest/MediatRTests/SourcesTests/UpdateCategoryTest.cs

```diff
@@ -0,0 +1,197 @@
+﻿using AutoMapper;
+using Moq;
+using Streetcode.BLL.DTO.Sources;
+using Streetcode.BLL.Interfaces.BlobStorage;
+using Streetcode.BLL.MediatR.Sources.SourceLink.Update;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using System.Linq.Expressions;
+using Xunit;
+using Streetcode.DAL.Entities.Media.Images;
+using Microsoft.EntityFrameworkCore.Query;
+using System.Threading.Tasks;
+using System.Threading;
+using Streetcode.BLL.Interfaces.Logging;
+using Microsoft.Extensions.Localization;
+using Streetcode.BLL.SharedResource;
+using MediatR;
+using Streetcode.BLL.MediatR.Sources.SourceLink.Create;
+
+namespace Streetcode.XUnitTest.MediatRTests.SourcesTests
+{
+    public class UpdateCategoryTests
+    {
+        private Mock<IRepositoryWrapper> _mockRepository;
+        private Mock<IMapper> _mockMapper;
+        private readonly Mock<ILoggerService> _mockLogger;
+        private readonly Mock<IStringLocalizer<FailedToUpdateSharedResource>> _mockLocalizerFailedToUpdate;
+        private readonly Mock<IStringLocalizer<CannotConvertNullSharedResource>> _mockLocalizerConvertNull;
+
+        public UpdateCategoryTests()
+        {
+            _mockRepository = new();
+            _mockMapper = new();
+            _mockLogger = new Mock<ILoggerService>();
+            _mockLocalizerConvertNull = new Mock<IStringLocalizer<CannotConvertNullSharedResource>>();
+            _mockLocalizerFailedToUpdate = new Mock<IStringLocalizer<FailedToUpdateSharedResource>>();
+        }
+
+        [Theory]
+        [InlineData(1)]
+        public async Task ShouldReturnSuccessfully_WhenUpdated(int returnNumber)
+        {
+            // Arrange
+            var testCategory = GetCategory(1);
+            var testCategoryDTO = GetCategoryDTO();
+
+            SetupUpdateRepository(returnNumber);
+            SetupMapper(testCategory, testCategoryDTO);
+            SetupImageRepository();
+
+            var handler = new UpdateCategoryHandler(_mockRepository.Object, _mockMapper.Object, _mockLogger.Object, _mockLocalizerFailedToUpdate.Object, _mockLocalizerConvertNull.Object);
+
+            // Act
+            var result = await handler.Handle(new UpdateCategoryCommand(testCategoryDTO), CancellationToken.None);
+
+            // Assert
+            // Assert
+            Assert.True(result.IsSuccess);
+            Assert.Empty(result.Errors); // Перевірка відсутності помилок
```

Comment:
please remove there `Assert` comments here, as it is clear what these `Asserts` do
Also it is better to write comments in English language since it is more common